common/shlibs

```diff
@@ -592,7 +592,7 @@ libecpg_compat.so.3 postgresql-libs-8.4.2_1
 libecpg.so.6 postgresql-libs-8.4.2_1
 libpgtypes.so.3 postgresql-libs-8.4.2_1
 libpq.so.5 postgresql-libs-8.4.2_1
-libmypaint-1.3.so.0 libmypaint-1.3.0_1
+libmypaint-1.5.so.1 libmypaint-1.5.0_1
```

Comment:
probably revdeps are broken due to this, see `xbps-query -RX libmypaint`

Reply:
So I should keep the old entry and just add the new one?

Reply:
No. I think you'll need to bump revision numbers of packages that you see in `xbps-query -RX libmypaint`

Reply:
OK, added the revision bumps. Checks fail due to timeout but building works fine on my system.